frontend/src/app/projects/models/model-source/t4c/manage-t4c-model/manage-t4c-model.component.ts

```diff
@@ -79,105 +61,118 @@ export class ManageT4CModelComponent implements OnInit, OnDestroy {
     private toastService: ToastService
   ) {}
 
+  public form = new FormGroup({
+    t4cInstanceId: new FormControl<number | null>(null, Validators.required),
+    t4cRepositoryId: new FormControl<number | null>(
```

Comment:
Short note to myself: We should align if we want to use `ID` or `Id`. What's your opinion?

Reply:
I prefer to use `Id`, but that's more of a personal preference than any real reason. So it would be fine to use `ID` as well.

Reply:
Ok, then let's use `Id` in the future 👍